tune_sklearn/tune_basesearch.py

```diff
@@ -292,13 +299,13 @@ def _fit(self, X, y=None, groups=None, **fit_params):
         if self.n_jobs is not None:
             if self.n_jobs < 0:
                 resources_per_trial = {
-                    "cpu": max(multiprocessing.cpu_count() + 1 + self.n_jobs,
-                               1),
+                    "cpu": max((multiprocessing.cpu_count() + 1 + self.n_jobs)/self._n_trials,
```

Comment:
Do we need to divide everything by `n_trials` actually? I think this would actually make most sense/avoid bugs but the default case (when `n_jobs=None`) would be to use 1 cpu for the entire fitting process, even if the machine has more cores to use. Would it be better to default to cpus available/n_trials?

Reply:
if n_jobs == -1, then "cpu": 1 IMO